gbpservice/nfp/configurator/lib/demuxer.py

```diff
@@ -105,45 +103,56 @@ def get_service_agent_info(self, operation, service_type, request_data):
         """
 
         sa_info_list = []
+        vendor_map = {const.FIREWALL: const.VYOS,
```

Comment:
Why is vendor map hard coded? Can't this be prepared base on what drivers got loaded?